controllers/packetcluster_controller.go

```diff
@@ -132,13 +132,13 @@ func (r *PacketClusterReconciler) reconcileNormal(ctx context.Context, clusterSc
 	default:
 		// If there is an ElasticIP with the right tag just use it again
 		clusterScope.PacketCluster.Spec.ControlPlaneEndpoint = clusterv1.APIEndpoint{
-			Host: ipReserv.Address,
+			Host: *ipReserv.Address,
```

Comment:
For `nil` safe accessors, we could rely on the generated getters: `ipReserv.GetAddress()`.

On the other hand, it may be harder to debug the odd case that ipReserv was nil and APIEndpoint received "" as the Host.